src/RealtimeEffectPanel.cpp

```diff
@@ -842,23 +1204,32 @@ class RealtimeEffectListWindow : public wxScrolledWindow
    void ReloadEffectsList()
    {
       wxWindowUpdateLocker freeze(this);
+      
+      const auto hadFocus = mEffectListContainer->IsDescendant(FindFocus());
       //delete items that were added to the sizer
+      mEffectListContainer->Hide();
       mEffectListContainer->GetSizer()->Clear(true);
 
-#ifdef __WXGTK__
-      //Workaround for GTK: Underlying GTK widget does not update
-      //its size when wxWindow size is set to zero
+      
       if(!mTrack || RealtimeEffectList::Get(*mTrack).GetStatesCount() == 0)
          mEffectListContainer->Hide();
-#endif
-
+      
+      auto isEmpty{true};
```

Comment:
Possibly change to 'const auto'.  Just being pedantic.

Reply:
Can't be made const, it changes value later

Reply:
Ah, I missed that.